keys.go

```diff
@@ -23,8 +24,36 @@ import (
 
 var RSAKeySize = 4096
 
+func getDatabasePath() string {
```

Comment:
This doesn't completely work. We use this function before we ever create all the directories. We should instead only return `"/usr/share/secureboot"` if it exists, and in all other cases return `"/etc/secureboot"`

Reply:
It works. I tested it on my machine without any directory existing.

Right now it works in this order:
1. `/etc/secureboot` if it exists
2. `/usr/share/secureboot` if it exists
3. `/etc/secureboot` if neither exists

> We should instead only return "/usr/share/secureboot" if it exists, and in all other cases return "/etc/secureboot"

So if both locations exists you want it `/usr` to have higher priority over `/etc`